bigchaindb/tendermint/event_stream.py

```diff
@@ -13,7 +13,7 @@
 
 HOST = getenv('BIGCHAINDB_TENDERMINT_HOST', 'localhost')
 PORT = int(getenv('BIGCHAINDB_TENDERMINT_PORT', 46657))
-URL = f'ws://{HOST}:{PORT}/websocket'
+URL = 'ws://{HOST}:{PORT}/websocket'
```

Comment:
I couldn't understand this, why don't we need formating?

Reply:
This is a syntax error no? flake8 was also failing.

Reply:
f-strings were added in Python 3.6:

https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep498

Reply:
@muawiakh My query was, do variables get replaced without using `f`? because I think without `f` value of `URL` will be `'ws://{HOST}:{PORT}/websocket'`

Reply:
Makes sense, now. I didn't know about that, I will update again.